layers/+lang/haskell/packages.el

```diff
@@ -67,6 +67,7 @@
     :defer t
     :config
     (progn
+      (spacemacs/set-leader-keys-for-major-mode 'haskell-mode "rs" 'intero-apply-suggestions)
```

Comment:
I would move it to [this function](https://github.com/Tehnix/spacemacs/blob/064650b9421efb90be9f0e7b3d2c30acd759876c/layers/%2Blang/haskell/funcs.el#L49). Now we support several completion backends and `spacemacs-haskell//setup-completion-backend` calls proper setup function whenever `haskell` source file is opened. This flow allows user to change completion backend on per-project basis. 

This key binding doesn't conflict with key bindings from other completions... but who knows if it will in the future? :)